### PR TITLE
BUG 286987:[advertising-api-build][store-api-build][winrt-api-build]p…

### PR DESCRIPTION
**Pull request by v-haiboz**
…age html diff: v2 set inlineeditbutton while v3 doesn't

### CODE REVIEW

File: ECMA2Yaml/ECMAHelper/ECMA2YamlConverter.cs [hunk 1]
```diff
@@ -68,6 +68,7 @@ public class ECMA2YamlConverter
             }
 
             Console.WriteLine("Building loaded files...");
+            Console.WriteLine($"ECMA2YamlRepoConfig:{config}");
```
Comment (TianqiZhang) [inline ECMA2Yaml/ECMAHelper/ECMA2YamlConverter.cs]: we can just json serialize this object for output

Comment (v-haiboz) [reply ECMA2Yaml/ECMAHelper/ECMA2YamlConverter.cs]: Updated

File: ECMA2Yaml/ECMAHelper/YamlConverters/SDPYamlConverter.cs [hunk 2]
```diff
@@ -80,7 +80,28 @@ public void Convert()
                 }
             }
         }
+        public static ReflectionItem HandleContentSourceMeta(ReflectionItem item)
```
Comment (TianqiZhang) [inline ECMA2Yaml/ECMAHelper/YamlConverters/SDPYamlConverter.cs]: can this be shared with existing method in ECMAStore?

Comment (v-haiboz) [reply ECMA2Yaml/ECMAHelper/YamlConverters/SDPYamlConverter.cs]: Updated

File: ECMA2Yaml/ECMAHelper/ECMA2YamlRepoConfig.cs [hunk 3]
```diff
@@ -21,5 +21,6 @@ public class ECMA2YamlRepoConfig
 
         [JsonProperty("UWP")]
         public bool UWP { get; set; }
+
```
Comment (TianqiZhang) [inline ECMA2Yaml/ECMAHelper/ECMA2YamlRepoConfig.cs]: extra line here

File: ECMA2Yaml/ECMAHelper/ECMA2YamlConverter.cs [hunk 4]
```diff
@@ -68,9 +70,23 @@ public class ECMA2YamlConverter
             }
 
             Console.WriteLine("Building loaded files...");
+            Console.WriteLine($"ECMA2YamlRepoConfig:{JsonConvert.SerializeObject(config)}");
             store.UWPMode = config?.UWP ?? false;
             store.Build();
 
+            if (store.UWPMode)
```
Comment (TianqiZhang) [inline ECMA2Yaml/ECMAHelper/ECMA2YamlConverter.cs]: we don't need to check for UWPMode here, we didn't check in previous version

File: ECMA2Yaml/ECMAHelper/Models/ECMAStore.cs [hunk 5]
```diff
@@ -149,7 +149,47 @@ public bool TryGetTypeByFullName(string fullName, out Type type)
...
+                item.SourceDetail = new GitSourceDetail()
+                {
+                    Path = mdPath,
+                    RepoBranch = publicGitBranch,
+                    RepoUrl = publicGitRepoUrl
+                };
+                item.Metadata.Remove("contentSourcePath");
+            }
+        }
         public void TranslateSourceLocation(
```
Comment (TianqiZhang) [inline ECMA2Yaml/ECMAHelper/Models/ECMAStore.cs:193]: we can call `TranlateContentSourceMeta` directly from this method.

Comment (v-haiboz) [reply ECMA2Yaml/ECMAHelper/Models/ECMAStore.cs:193]: for "ECMA2YamlConverter.Run()",this method has redundant parameters and logic, and it is only called in the program.Main.

File: ECMA2Yaml/ECMAHelper/ECMA2YamlConverter.cs [hunk 6]
```diff
@@ -19,6 +19,8 @@ public class ECMA2YamlConverter
             Action<LogItem> logWriter = null,
             string logContentBaseDirectory = null,
             string sourceMapFilePath = null,
+            string repoUrl = null,
```
Comment (TianqiZhang) [inline ECMA2Yaml/ECMAHelper/ECMA2YamlConverter.cs]: `publicRepoUrl` and `publicRepoBranch`

File: ECMA2Yaml/ECMAHelper/ECMA2YamlRepoConfig.cs (hunk 3 above)
Comment (v-haiboz) [reply ECMA2Yaml/ECMAHelper/ECMA2YamlRepoConfig.cs]: Updated

File: ECMA2Yaml/ECMAHelper/ECMA2YamlConverter.cs (hunk 4 above)
Comment (v-haiboz) [reply ECMA2Yaml/ECMAHelper/ECMA2YamlConverter.cs]: removed

**Review by TianqiZhang — APPROVED** [review]

**Pull request merged**